### PR TITLE
ceph-build: check chacra for ceph-osd package to test for "should I build"

### PR DESCRIPTION
**Pull request by dmick**

### CODE REVIEW

**Comment by dmick** [comment]
I would like to find a good way to test this without merging to master, but I don't know what it would be

File: ceph-build/build/build_deb [hunk 1]
```diff
@@ -57,6 +57,9 @@ gen_debian_version() {
 
 bpvers=`gen_debian_version $debian_version $DIST`
 
+# look for a specific package to tell if we can avoid the build
+chacra_endpoint="ceph/${chacra_ref}/${distro}/${DIST}/${ARCH}/ceph-osd_${vers}-${bpvers}_${ARCH}.deb"
```
Comment (alfredodeza) [inline ceph-build/build/build_deb]: using `ceph-osd` here would make this check not work for Ceph versions where the split didn't exist (Hammer?)

**Comment by ktdreyer** [comment]
> I would like to find a good way to test this without merging to master, but I don't know what it would be

The way I typically do this is that I run JJB's `jenkins-jobs` by hand on my local system, so that I've applied the change to jenkins.ceph.com prior to merging in GitHub. You can just modify the production "ceph-build" job this way, or maybe rename it... although using a renamed job is really hard.

It would be nice to have a staging environment for this sort of thing.

File: ceph-build/build/build_deb (hunk 1 above)
Comment (dmick) [inline ceph-build/build/build_deb]: argh, totally missed this comment.  OK, can change to librados2

**Pull request merged**